nservicebus/containers/index_content_core_[,5].partial.md

```diff
@@ -5,6 +5,7 @@ Note that the instance of `IBus` is scoped for the lifetime of the container. He
 
 ### Cleanup
 
-When using an external container normally the bus instance is not disposed of manually. If `IBus.Dispose()` is called that would indirectly trigger the container to dispose lifetime scope.
+When using an external container normally the bus instance is not disposed of automatically. The following dispose sequence should be applied if latest container adapter packages are used:
```

Comment:
I had a hard time when I wrote this. First I didn't mention it at all, then I figured it might be helpful because we can't assume people run the latest bugfix. We would need to specify the version which contains the change for all container packages. Should we do it?